compiler/extccomp.nim

```diff
@@ -27,7 +27,9 @@ type
     hasGcGuard,               # CC supports GC_GUARD to keep stack roots
     hasGnuAsm,                # CC's asm uses the absurd GNU assembler syntax
     hasNakedDeclspec,         # CC has __declspec(naked)
-    hasNakedAttribute         # CC has __attribute__((naked))
+    hasNakedAttribute,        # CC has __attribute__((naked))
```

Comment:
I think it suffices to distinguish between __attribute__ and __declspec at this point.